style.css

```diff
@@ -2,9 +2,92 @@
 body{
   margin: 0;
   padding: 0;
+  background-color: rgba(0, 0, 0, 0.705);
+}
+#title{
+  font-weight: 700;
+  line-height: 1.2;
+  color: white;
+
+}
+#description{
+  font-style: italic;
+  font-weight: 200;
+  text-shadow: 1px 1px 1px rgb(0 0 0 / 40%);
+  color: white;
 }
 .container{
   display: flex;
   flex-direction: column;
-  justify-items: center;
+  align-items: center;
+}
+#survey-form {
+   display: flex;
+   flex-direction: column;
+   gap: 10px;
+   width: 60%;
+   background-color: rgba(27, 27, 50,0.8);
+   color: #fff;
+   padding: 5% 2%;
+   border-radius:10px;
+}
+label {
+  align-self:flex-start;
+  font-size: 20px;
+  font-weight: bold;
+  margin-left: 5%;
+  margin-top: 2%;
+}
+.text-field {
+   height: 30px;
+   margin: 0 5%;
+   border-radius: 5px;
+}
+.checkbox{
+  font-size: 20px;
+  font-weight: normal;
+}
+.message{
+  font-weight: 15px;
+  font-size: small;
+}
+.radio-btn{
+  margin-left: 5%;
+}
+#comment{
+  height: auto;
+  margin: 0 5%;
+  border-radius: 5px;
+}
+/* #submit{
+  background-color: green;
+  color:#fff;
+} */
```

Comment:
- [ ] Please remove the committed old pieces of code as inline comments to make your code neater.